tests/cfour/default.nix

```diff
@@ -0,0 +1,22 @@
+{ batsTest, cfour } :
+
+batsTest {
+  name = "cfour";
+
+  nativeBuildInputs = [ cfour ];
+
+  auxFiles = [
+    ./ZMAT
+    ./GENBAS
+  ];
+
+  outFile = [ "cfour.out" ];
+
+  OMP_NUM_THREADS=2;
+
+  testScript = ''
+    @test "Run CFour" {
+      xcfour > cfour.out
```

Comment:
In such tests it is always nice to have check the final energy. That we have a confirmation that it not only runs but always  yields the same value.

Reply:
Done!